### ISSUE TITLE
Minimuxer is not ready. Ensure you have WiFi and vpn set up.

### ISSUE THREAD

**Issue by 3van0910** [khanhduytran0/SparseBox#63]
I’m on IOS 18.2 dev beta 2 and I can’t get the app to work. I tried waiting in between tries, and tried turning off and on my vpn but it’s not working. Please help. I have an iPhone 13

**Comment by ChrisGleich1AufsMaul** [comment]
Did you select a pairing file?

Right now I´m stuck at Error "Unsupported command '/var/mobile/ Containers/Data/Application/57FD2533-D611-4EAA-ADA1-5BE3B5F0A663/Documents/'.
Usage: idevicebackup2 [OPTIONS] CMD
[CMDOPTIONS] DIRECTORY".

**Comment by 3van0910** [comment]
> Did you select a pairing file?
> 
> 
> 
> Right now I´m stuck at Error "Unsupported command '/var/mobile/ Containers/Data/Application/57FD2533-D611-4EAA-ADA1-5BE3B5F0A663/Documents/'.
> 
> Usage: idevicebackup2 [OPTIONS] CMD
> 
> [CMDOPTIONS] DIRECTORY".

Yeah, I even deleted the pairing file and got a new one to see if it would work but I still have the problem
